src/modules/users/model.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '../../loaders/sequelize';
+
+export interface UserModel extends Model {
+    id: string;
+    login: string;
+    password: string;
+    age: number;
+    isDeleted: boolean;
+}
+
+export const User = sequelize.define<UserModel>('Users', {
```

Comment:
when defining model, the name should be singular noun, i.e. instead of `Users` it should be `User`

Reply:
done